.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: php
+php:
+  - 5.4
+  - 5.5
+  - 5.6
+  - '7'
+  - hhvm
+script:
+  - script/test
+before_install:
+  - composer self-update --no-interaction
+before_script:
+  - composer install --no-interaction
```

Comment:
This is mostly in `script/test` already, just without the `--no-interaction` bit.